### ISSUE TITLE
Error when running JqxList on a yaml file with `/` in the key

### ISSUE THREAD

**Issue by dkarter** [gennaro-tedesco/nvim-jqix#27]
Thank you for this plugin! I just discovered it and it looks very useful.

I just bumped into an error when running `JqxList` for the first time on a yaml file with `/` characters in the keys:

![CleanShot 2023-02-23 at 22 38 51@2x](https://user-images.githubusercontent.com/551858/221093329-4615d272-ccfc-420b-8eea-6bfe97a5ccf8.png)

```
E5108: Error executing lua .../nvim/site/pack/packer/opt/nvim-jqx/lua/nvim-jqx/jqx.lua:21: Vim:E492:
 Not an editor command: TrailingEmptyLines/echo line('.')
stack traceback:
        [C]: in function 'nvim_exec'
        .../nvim/site/pack/packer/opt/nvim-jqx/lua/nvim-jqx/jqx.lua:21: in function 'get_key_locatio
n'
        .../nvim/site/pack/packer/opt/nvim-jqx/lua/nvim-jqx/jqx.lua:64: in function 'populate_qf'
        ...nvim/site/pack/packer/opt/nvim-jqx/lua/nvim-jqx/init.lua:52: in function 'jqx_open'
        [string ":lua"]:1: in main chunk
```

Here's the file:

```yaml
AllCops:
  NewCops: enable
  TargetRubyVersion: 3.1

Metrics/ClassLength:
  Enabled: false

Metrics/MethodLength:
  Enabled: false

Metrics/AbcSize:
  Enabled: false

Layout/TrailingEmptyLines:
  Enabled: false

Style/TrailingCommaInArrayLiteral:
  EnforcedStyleForMultiline: comma

Style/TrailingCommaInHashLiteral:
  EnforcedStyleForMultiline: comma

Style/TrailingCommaInArguments:
  EnforcedStyleForMultiline: comma
```

FWIW it seems like `yq` is able to parse this file

<img width="397" alt="CleanShot 2023-02-23 at 22 45 56@2x" src="https://user-images.githubusercontent.com/551858/221094082-dd31fa67-c843-4d02-bfda-b6f9a9e58ebb.png">

**Comment by gennaro-tedesco** [comment]
Good morning and thank you for reporting this issue: the error is generated because I am not escaping `/` (or other special characters for that matter) when looking up the keys location in the current buffer. It should be a relatively easy fix :)

**Comment by gennaro-tedesco** [comment]
I have opened https://github.com/gennaro-tedesco/nvim-jqx/pull/28 and after testing it on your file it seems to do the job. Would you help me test it yourself as well (another pair of eyes is always useful :p)? You could specify the branch `fix_escaping` in your plugin manager (most plugin managers support installing from different branches) and run your workflow as usual to make sure all is good.

**Comment by dkarter** [comment]
Thank you for the quick fix! this seems to be working well now. When testing out the branch I did notice that the `yq` repository referenced in the readme was pointing at an incorrect distribution of `yq` (Go implementation instead of Python implementation that this plugin supports). I opened a PR to correct the readme #29

**Comment by gennaro-tedesco** [comment]
Addressed by [v0.1.3](https://github.com/gennaro-tedesco/nvim-jqx/releases/tag/v0.1.3). Updating the plugin to the latest release will work now.
